railties/lib/rails/generators/rails/app/templates/bin/update

```diff
@@ -0,0 +1,23 @@
+require 'pathname'
+require 'fileutils'
+include FileUtils
+
+# path to your application root.
+APP_ROOT = Pathname.new File.expand_path('../../', __FILE__)
+
+chdir APP_ROOT do
+  # This script is a way to update your application automatically
+  # Add necessary update steps to this file.
+
+  puts '== Installing dependencies =='
+  system 'bundle install'
```

Comment:
It might be interesting to take in consideration the `system!` implementation added by other PR, to abort in case of a failure.